modules/gitops-document-attributes.adoc

```diff
@@ -0,0 +1,12 @@
+// Standard document attributes to be used in the documentation
+//
+// The following are shared by all documents:
+:toc:
+:toclevels: 4
+:experimental:
+//
+// Product content attributes, that is, substitution variables in the files.
+//
+:gitops-title: Red Hat OpenShift GitOps
+:gitops-shortname: GitOps
+:gitops-ver: release-tech-preview-4.7
```

Comment:
confirm what this version is called. Thinking it would be 1.0 since it's the first GitOps version.